docs/tutorial/multi_car_paint.ipynb

```diff
@@ -0,0 +1,541 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "i9nXiSl26DJK"
+   },
+   "source": [
+    "# Quantum Alternating Operator Ansatz for Two-Color Multi Car Paint Shop Problem\n",
+    "The goal of the multi-car paint shop problem is to reduce the number of color switches between cars in a paint shop line during the manufacturing process, which is recognized as an NP-hard problem.\n",
+    "\n",
+    "Let's consider two-color scenario in this tutorial. \n",
+    "\n",
+    "\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Expressing the problem using a mathematical model\n",
+    "\n",
+    "Given a set of cars $ X = \\{x_1, x_2, \\dots, x_n\\} $ that need to be painted in one of two colors, denoted as $ x_i=0 $ and $ x_i=1 $,  for each $ i = 1, \\dots n$ . The objective is to minimize the number of color switches (i.e., changes from $ 0 $ to $ 1 $ or from $ 1 $ to $ 0 $) between consecutive cars in the sequence.\n",
+    "\n",
+    "\n",
+    "The **cost** of the sequence is the number of times the color switches between consecutive cars, which can be formulated as:\n",
+    "\n",
+    "$$\n",
+    "\\text{min} \\quad -\\sum_{i=0}^{N-2} (x_i - 0.5) \\cdot (x_{i+1} - 0.5)\n",
```

Comment:
This formulation has huge jump because the spin variables $s_i \in \{-1,1\}$ are used in the original formulation.
However, in jijmodeling, we can only use binary varaibles $x_i \in \{0,1\}$, so we use this weird formulation, so I think it it better to start the explanation from spin variables, then change the spin variables into binary variables.